README.md

```diff
@@ -1 +1,176 @@
-# angular-link-dom-manipulation-readme
\ No newline at end of file
+# Using the link function for DOM manipulation
+
+## Overview
+
+We've had a look at the link function, now lets do some DOM manipulation with it.
+
+## Objectives
+
+- Use a native event API
+- Use a Controller inside Directive
+- Update Controller layer using native events
+
+## Native Events
+
+Now that we've got access to the actual DOM nodes, we can do our own, manual DOM events on the elements. Whilst generally we'd use `ng-click` and let Angular do the dirty work for us, we can do it ourselves manually if we are using 3rd-party plugins that aren't compatible with Angular.
```

Comment:
Is the example below more or less how ng-click is implemented? If so, tell the reader this - let them know that we're taking a deeper look at how the built-in directives are implemented.